src/js/pre-need/config/form.js

```diff
@@ -43,6 +49,8 @@ const {
   preneedAttachments
 } = fullSchemaPreNeed.properties.application.properties;
 
+veteran.properties.serviceRecords.items.properties.serviceBranch.enum = ['AL', 'CC', 'FS', 'FT', 'ES', 'CM', 'C3', 'C2', 'C4', 'C7', 'C5', 'GS', 'CI', 'FP', 'CS', 'CV', 'XG', 'CB', 'FF', 'GP', 'MO', 'NO', 'NN', 'NM', 'PA', 'PG', 'KC', 'PS', 'RO', 'CF', 'CE', 'AF', 'XF', 'AG', 'AR', 'AC', 'AA', 'AT', 'NG', 'XR', 'CO', 'CA', 'GC', 'CG', 'XC', 'MC', 'MM', 'XA', 'CD', 'PH', 'GU', 'WP', 'WA', 'WS', 'WR'];
```

Comment:
Is this necessary? I thought it was in the schema?

Reply:
You are correct @jbalboni, this can be removed.